examples/lora/lorawan/basic-demo/main.go

```diff
@@ -81,14 +83,29 @@ func main() {
 	// Connect the lorawan with the Lora Radio device.
 	lorawan.UseRadio(radio)
 
+	lorawan.UseRegionSettings(region.EU868())
+
 	// Configure AppEUI, DevEUI, APPKey
 	setLorawanKeys()
 
+	if debug {
```

Comment:
Change this conidtional to:

```go
if Debug != ""
```

If you like the idea of passing in the `Debug` using ldflags.

Reply:
Thanks, I didn't know this ldflags trick. 

As it works with lowercase, I didn't change the current name of the variable (debug)

I tested the feature with : 

$ tinygo build -target=lorae5 (no logs)
$ tinygo build -ldflags="-X 'main.debug=true'" -target=lorae5 (debugs logs)

Reply:
Sounds good to me!